pyproject.toml

```diff
@@ -95,3 +81,45 @@ torchseg = ["py.typed"]
 
 [tool.setuptools.packages.find]
 include = ["torchseg*"]
+
+[tool.ruff]
+exclude = [
```

Comment:
By default, everything in .gitignore is already ignored, so you probably don't need to override this: https://docs.astral.sh/ruff/settings/#respect-gitignore